ops/services/alerts/app_service_metrics/exceptions.tf

```diff
@@ -162,6 +162,41 @@ ${local.skip_on_weekends}
   }
 }
 
+resource "azurerm_monitor_scheduled_query_rules_alert" "all_experian_auth_failures" {
+  name                = "${var.env}-all-experian-auth-failures"
+  description         = "${local.env_title} alert when two or more ExperianAuthExceptions are seen"
+  location            = data.azurerm_resource_group.app.location
+  resource_group_name = var.rg_name
+
+  action {
+    action_group = var.action_group_ids
+  }
+
+  data_source_id = var.app_insights_id
+  enabled        = contains(var.disabled_alerts, "experian_auth_failures") ? false : true
+
+  query = <<-QUERY
+requests
+${local.skip_on_weekends}
+| where timestamp > ago(2h) and success == false
+| join kind= inner (
+    exceptions
+    | where timestamp > ago(2h)
+    )
+    on operation_Id
+| where type hassuffix "ExperianAuthException"
+| project timestamp, exceptionType = type, failedMethod = method, requestName = name
+  QUERY
+
+  severity    = 1
+  frequency   = 5
+  time_window = 5
+  trigger {
+    operator  = "GreaterThan"
+    threshold = 1
```

Comment:
I _think_ when an ExperianAuthException is triggered, it generates two rows, so this threshold might need to be 2. double-checking in the Azure logs

Reply:
yup so ExperianAuthException gets logged twice in the exceptions table when one occurs, so by nature of this join we end up with two rows in the resulting table. so I think the threshold should be raised to 2, or alternatively we could change the join type or group by `operationId` or something